bin/build-dps

```diff
@@ -17,4 +17,4 @@ set -euo pipefail
 
 basedir=$(dirname "$(readlink -f "$0")")
 
-"${basedir}/build" --no-dev
+AWS_PROFILE=maap-data-reader "${basedir}/build" --no-dev
```

Comment:
We need to run the build with the AWS_PROFILE as well?

Reply:
Actually, upon further experimentation, I don't think we need to specify the profile in either place. Let me remove both appearances and reregister and rerun to confirm.

Reply:
@jjfrench, everything worked without the explicit profile.